server/src/entities/exif.entity.ts

```diff
@@ -95,6 +95,9 @@ export class ExifEntity {
   @Column({ type: 'integer', nullable: true })
   bitsPerSample!: number | null;
 
+  @Column({ type: 'integer', nullable: true })
```

Comment:
should this to a decimal and allow values like 1.5, 2.5, etc?

Reply:
I looked at the exif-specs. It should be Integer.